data/human/human missions.txt

```diff
@@ -7453,3 +7453,775 @@ mission "Adelita 5"
+				decline
+			label studyskadeweland
+			action
+				outfit "Tear of Skade" -1
+				event "welandwait3skade" 90
+			`	Weland grins. "A little time's better than no time. Put 'er there."`
+			`	He takes your hand and shakes it with great enthusiasm. "Imagine what secrets await."`
+			`	Later on you transfer over the stone, 'Tear of Skade', to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
+				decline
+			label studysurtrweland
+			`	Weland grins. "A little time's better than no time. Put 'er there."`
+			`	He takes your hand and shakes it with great enthusiasm. "Imagine what secrets await."`
+			`	Later on you transfer over the stone, 'Blood of Surtr', to a feverishly eager Weland. You promise to back in three months, for stone and payment.`
+			action
+				event "welandwait3surtr" 90
+				outfit "Blood of Surtr" -1
+
+mission "Stone Harmonization a"
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination "Wayfarer"
+	landing
+	to offer
+		has "Dragon Tech 1: offered"
+		has "event: welandstone"
+		random < 10
+	on offer
+		require "Tear of Skade"
+		require "Blood of Surtr"
+		dialog
+			`Your inbox blinks with a new message from Weland on Wayfarer.`
+			`	<name>, I was wanting to speak with you about something.`
+			`	Please stop by my shop on Wayfarer at your earliest convenience.`
+			`				-Xeno-Smith`
+			`					Weland`
+	on complete
+		conversation
+			`You find Weland tinkering away in his shop. He puts his inscrutable tools down and waves to you as you enter.`
+			`	"Ah, there you are. I am sure you're wonderin' yourself sick trying to divine why I called ya." He grins, and limps toward you, dragging metal chains behind his feet.`
+			`	"I heard word, from someone who's done crew work on your ship, that you're in possession of a couple of Deep Stones." He nearly whispers the last the two words.`
+			`	"I've been told you call 'em, the 'Tear of Skade', and the 'Blood of Surtr'". He watches your eyes carefully as he names off the stones. "Now, if they are what I think they are, well, they're worth a lot of money. I never met nobody that has laid eyes on a Deep Stone, let alone owned one. It's phantom tech, kind of stuff you hear about from spooks, and conspiracy nuts. Most say it isn't even real, and if it is, only a few ever been found, and those the government holds under lock and key.`
+			`	"But, here you are, flying free with not one, but two, on your ship, doin' whatever it is these strange stones do. I'd offer to buy one of 'em off ya, but I don't have anywhere nears enough cash for one of those.`
+			`	"So, what if I borrowed one of the stones? Just, hear me out, alright. Give me three months to study it, learn what secrets from it I can. I'd pay you 3 million credits for the pleasure. After 3 months, you come collect the stone, and your payment. Everyone wins. Whatd'ya say?"`
```

Comment:
The way this is written, it sounds like there was a possibility not everybody would be happy.
Maybe remove "Everyone wins" all together.

Reply:
It's a sales pitch, though. He's trying to convince you to do it.